src/applications/personalization/profiles-wrapper/ProfilesWrapper.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Suspense } from 'react';
+import { connect } from 'react-redux';
+import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+import Profile2Wrapper from 'applications/personalization/profile-2/components/Profile2Wrapper';
+import ProfileOneWrapper from 'applications/personalization/profile360/containers/VAProfileApp';
+import environment from 'platform/utilities/environment';
+import routes from 'applications/personalization/profile-2/routes';
+import { isLOA3 as isLOA3Selector } from 'platform/user/selectors';
+import { selectShowProfile2 } from 'applications/personalization/profile-2/selectors';
+
+const LoadingPage = <Profile2Wrapper>Loading...</Profile2Wrapper>;
+
+const ProfilesWrapper = ({ showProfile2, isLOA3 }) => {
+  if (!showProfile2) {
+    return <ProfileOneWrapper />;
+  }
+
+  return (
+    <BrowserRouter>
+      <Suspense fallback={LoadingPage}>
+        <Switch>
+          {routes.map(route => {
+            if (route.requiresLOA3 && !isLOA3) {
+              return (
+                <Redirect
+                  from={route.path}
+                  key="/profile/personal-information"
+                  to="/profile/personal-information"
+                />
+              );
+            }
+
+            const Component = route.component;
+
+            return (
+              <Route
+                component={props => (
+                  <Profile2Wrapper {...props}>
+                    <Component />
+                  </Profile2Wrapper>
+                )}
+                exact
+                key={route.path}
+                path={route.path}
+              />
+            );
+          })}
+
+          <Redirect
```

Comment:
We want to redirect the user to `/profile/personal-information` when they land on `/profile`. If desired, it should be easy to add other redirects here based on LOA levels.

Reply:
So could we redirect to personal info if they are LOA3 but otherwise redirect to account security?